module/VuFind/tests/unit-tests/src/VuFindTest/View/Helper/Root/RecordDataFormatterTest.php

```diff
@@ -244,10 +245,25 @@ protected function getFormatter($additionalConfig = [])
                 'RecordDataFormatter' => $recordDataFormatterConfig,
             ])
         );
+        $schemaOrgHelper = new \VuFind\View\Helper\Root\SchemaOrg(
+            new \Laminas\View\Helper\HtmlAttributes()
+        );
+        $specManager = $this->getMockBuilder(\VuFind\RecordDataFormatter\Specs\PluginManager::class)
+            ->disableOriginalConstructor()
+            ->getMock();
```

Comment:
This can be simplified to:

```suggestion
        $specManager = $this->createMock(\VuFind\RecordDataFormatter\Specs\PluginManager::class);
```

Reply:
Thanks, done